operate/pom.xml

```diff
@@ -94,19 +94,17 @@
     <postgres-test-container.version>1.17.6</postgres-test-container.version>
     <version.docker-java-api>3.3.6</version.docker-java-api>
     <version.httpclient>4.5.14</version.httpclient>
-    <version.elasticsearch7>7.17.18</version.elasticsearch7>
+    <version.elasticsearch7>7.17.19</version.elasticsearch7>
     <version.elasticsearch>8.9.2</version.elasticsearch>
-    <version.lucene>8.11.1</version.lucene>
+    <version.lucene>8.11.3</version.lucene>
     <version.error-prone>2.25.0</version.error-prone>
     <version.guava>33.0.0-jre</version.guava>
-    <version.guava.annotations>r03</version.guava.annotations>
```

Comment:
These two guava properties were duplicates that were present elsewhere in the pom file